### PR TITLE
Create test for the service StepExecution ClickCss

### PR DESCRIPTION
**Pull request by Igor-Proh**
1. Updated StepExecutionClickCss
2. Created StepExecutionClickCssTest
3. Add Mockito to pom

### CODE REVIEW

File: executor-service/src/test/java/executor/service/service/step/impl/StepExecutionClickCssTest.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    }
+
+    @AfterEach
+    public void tearDown() {
+        stepExecutionClickCss = null;
+        webDriver = null;
+    }
+
+    @Test
+    public void testGetStepAction() {
```
Comment (Lokankara) [inline executor-service/src/test/java/executor/service/service/step/impl/StepExecutionClickCssTest.java]: Remove this 'public' modifier, please:
JUnit5 test classes and methods should have default package visibility

File: executor-service/src/main/java/executor/service/service/step/impl/StepExecutionClickCss.java [hunk 2]
```diff
@@ -14,7 +14,7 @@ public String getStepAction() {
 
     @Override
     public void step(WebDriver webDriver, Step step) {
-        String cssSelector = step.getAction();
+        String cssSelector = step.getValue();
         WebElement webElement = webDriver.findElement(By.cssSelector(cssSelector));
```
Comment (Lokankara) [inline executor-service/src/main/java/executor/service/service/step/impl/StepExecutionClickCss.java]: You can click on an element without creating a new one:
```webDriver.findElement(By.cssSelector(cssSelector)).click();```

**Review by Lokankara — APPROVED** [review]

File: executor-service/src/main/java/executor/service/service/step/impl/StepExecutionClickCss.java (hunk 2 above)
Comment (Igor-Proh) [reply executor-service/src/main/java/executor/service/service/step/impl/StepExecutionClickCss.java]: I agree)

File: executor-service/src/test/java/executor/service/service/step/impl/StepExecutionClickCssTest.java (hunk 1 above)
Comment (Igor-Proh) [reply executor-service/src/test/java/executor/service/service/step/impl/StepExecutionClickCssTest.java]: Ohh. Thanks.

**Review by MazurenkoNick — APPROVED** [review]

**Pull request merged**